src/quemb/molbe/mbe.py

```diff
@@ -980,7 +978,6 @@ def initialize(
                 assert_never(int_transform)
         else:
             eri = None
-        logger.info(f"ERI transform time: {eritransform_timer.str_elapsed()}")
```

Comment:
To be clearer on my last comment, I would prefer it if things like this are still preserved. i.e., after ERI transformation, logger should print out how long it took. This is a metric we are often interested in.

Reply:
this could actually be added into the `timer.timeit` method. Then the developer does not have the deal with the clutter of logging statements.